release/rllib_tests/learning_tests/yaml_files/sac/sac-halfcheetahbulletenv-v0.yaml

```diff
@@ -19,7 +19,7 @@ sac-halfcheetahbulletenv-v0:
         tau: 0.005
         target_entropy: auto
         no_done_at_end: false
-        n_step: 3
+        n_step: 1
```

Comment:
Here, too, remove rollout_fragment_length (or set to "auto") further below entirely.